picoquic/picoquic.h

```diff
@@ -1485,6 +1486,24 @@ void picoquic_set_default_wifi_shadow_rtt(picoquic_quic_t* quic, uint64_t wifi_s
 */
 void picoquic_set_default_bbr_quantum_ratio(picoquic_quic_t* quic, double quantum_ratio);
 
+/* The experimental API `picoquic_set_feedback_loss_notification` allow applications
+* to turn on the "feedback lost" event notification. These events are
+* passed to the congestion control algorithm, allowing it to react
+* quickly to a temporary loss of connectivity, instead of waiting
+* for retransmission timers. Delay sensitive applications use this
+* feature to stop queuing more data when connectivity is lost,
+* and thus avoid the queues of less urgent data to delay
+* arrival of urgent real time frames when connectivity is restored.
+* On the other hand, this feature may lower the performance of
+* applications sending lots of data, and thus should only be
+* used when applications require it.
+* 
+* The `should_notify` should be set 1 to enable the feature, or to 0
+* to stop notifications. It is set by default to zero when a connection
+* is created.
+*/
```

Comment:
i wonder of we should  also add the interval of the feedback lost events. From the code, it is 2 * ack frequency + some delta .. Is that right understanding?

Reply:
Yes, it is set to twice the maximum delay between receiving a packet and sending an ack. This is a bit arbitrary. The factor 2 is meant to allow for the loss of an ACK, but that not ideal. We could do something complicated, e.g., measuring the actual interval between ACKs and comparing it to the sending interval, maybe doing something like average plus couple of deviations. But that's harder to describe, and probably error prone.

Reply:
I updated the comment to describe the different with RTO.